skeleton-navigation/src/my-app.css__if_css

```diff
@@ -0,0 +1,48 @@
+body {
```

Comment:
If I'm not mistaken, this styling is more or less the styling I had in one of my live coding videos. If so, it should perhaps be both cleaned up and trimmed down for a skeleton.

Reply:
you are right :), but I think minimal styling is fine to have it nice looking. Please share your thoughts.